### PR TITLE
[initramfs] Updated required tools for initramfs

### PR DESCRIPTION
**Pull request by antony-rheneus**
Signed-off-by: Antony Rheneus <arheneus@marvell.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: build_debian.sh [hunk 1]
```diff
@@ -176,6 +176,11 @@ sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/scripts/init-bottom/varlog
 sudo cp files/initramfs-tools/union-fsck $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/union-fsck
 sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/union-fsck
 pushd $FILESYSTEM_ROOT/usr/share/initramfs-tools/scripts/init-bottom && sudo patch -p1 < $OLDPWD/files/initramfs-tools/udev.patch; popd
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo cp files/initramfs-tools/arm-utils $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/arm-utils
+    sudo chmod +x $FILESYSTEM_ROOT/etc/initramfs-tools/hooks/arm-utils
```
Comment (lguohan) [inline build_debian.sh]: i think we need to differentiate the arm and the bootloader. this is not really related to arm but bootloader. I do not mind to install then in x86 build as long as they are not too large.

File: build_debian.sh [hunk 2]
```diff
@@ -499,8 +504,23 @@ fi
 ## Remove gcc and python dev pkgs
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc libpython2.7-dev
 
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install u-boot-tools mtd-utils
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-mark manual u-boot-tools mtd-utils
```
Comment (lguohan) [inline build_debian.sh]: I think we can installer them in x86 as well.

Comment (antony-rheneus) [reply build_debian.sh]: No, we cant. these packages are specific to arch other than x86/amd

File: build_debian.sh (hunk 1 above)
Comment (antony-rheneus) [reply build_debian.sh]: Ok will remove and change name to uboot-utils

File: build_debian.sh (hunk 2 above)
Comment (lguohan) [reply build_debian.sh]: i do not understand here. https://packages.debian.org/search?keywords=mtd-utils

Comment (lguohan) [reply build_debian.sh]: same for the u-boot-tools. https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=u-boot-tools

File: platform/marvell-armhf/sonic-platform-et6448m/debian/.debhelper/generated/sonic-platform-et6448m/installed-by-dh_install [hunk 3]
```diff
@@ -0,0 +1 @@
+./et6448m_plt_setup.sh
```
Comment (lguohan) [inline platform/marvell-armhf/sonic-platform-et6448m/debian/.debhelper/generated/sonic-platform-et6448m/installed-by-dh_install]: why do we check-in generated files?

Comment (lguohan) [reply platform/marvell-armhf/sonic-platform-et6448m/debian/.debhelper/generated/sonic-platform-et6448m/installed-by-dh_install]: same comment for the generated file below.

File: platform/marvell-armhf/sonic-platform-et6448m/debian/files [hunk 4]
```diff
@@ -0,0 +1,2 @@
+sonic-platform-et6448m_0.1_armhf.buildinfo unknown optional
+sonic-platform-et6448m_0.1_armhf.deb unknown optional
```
Comment (lguohan) [inline platform/marvell-armhf/sonic-platform-et6448m/debian/files]: this is generated file. do not check-in.

File: platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums [hunk 5]
```diff
@@ -0,0 +1,2 @@
+d4fe1b11c5ab1ca5f69e61d99958c0b2  usr/sbin/et6448m_plt_setup.sh
```
Comment (lguohan) [inline platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums]: generated file

File: build_debian.sh (hunk 2 above)
Comment (antony-rheneus) [reply build_debian.sh]: will update.

File: platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums (hunk 5 above)
Comment (antony-rheneus) [reply platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums]: Sorry, It was my mistake. will remove all generated files.

Comment (antony-rheneus) [reply platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums]: removed

**Comment by antony-rheneus** [comment]
vsimage failed due to 
"mv: cannot stat '/etc/sonic/minigraph.xml': No such file or directory""

Can you suggest what could be the issue?

**Comment by lguohan** [comment]
retest vsimage please
